### ISSUE TITLE
[Task ⚗] Invite button 

### ISSUE THREAD

**Issue by HueByte** [HueByte/Huppy#30]
## Description
Put **Add to Server** button to Huppy card

Example: 
![img](https://cdn.discordapp.com/attachments/1010987565781885079/1011575135540617246/unknown.png)

## Goals
- [ ] Add invite button to Huppy card
- [ ] Make it disablable (either hardcode for now or add property in config)

**Comment by leemustache** [comment]
[Think it needs to be done in the bot's discord settings, not in code](https://youtu.be/HZF6lMshuX0)

Seems easy enough though, can you change this setting and see if it works? @HueByte

**Comment by HueByte** [comment]
@LeeMustache you're right, I'm gonna keep the issue open as for now and I'll do it for Huppy once we make public release
